example/cmd/device-simple/Dockerfile

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (c) 2018, 2019 Intel
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+FROM golang:1.11-alpine AS builder
+
+ENV GO111MODULE=on
+WORKDIR /go/src/github.com/edgexfoundry/device-sdk-go
+
+LABEL license='SPDX-License-Identifier: Apache-2.0' \
+  copyright='Copyright (c) 2018, 2019: Intel'
+
+RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/repositories
+
+# add git for go modules
+RUN apk update && apk add make git
+
+COPY go.mod .
+
+RUN go mod download
```

Comment:
go mod download required https_proxy

Reply:
Yup, and then when I enabled proxy for this then communication between services fail... Proxy Hell!

Reply:
Have you tried using the build-arg I mention above to set this up? It should enable proxy only during build of the docker image. Since it doesn't leave an environment variable enabled within the container service communication doesn't get impacted.